k8s/wordpress/apptest/tester/tests/basic-suite.yaml.template

```diff
@@ -8,7 +13,42 @@ actions:
       bodyText:
         html:
           title:
-            matches: 'WordPress . Installation'
+            equals: 'WordPress on Google Kubernetes Engine – Just another WordPress site'
+
+- name: Site address should be 200 OK (external IP)
+  httpTest:
+    url: http://${EXTERNAL_IP}
+    expect:
+      statusCode:
+        equals: 200
+      bodyText:
+        html:
+          title:
+            equals: 'WordPress on Google Kubernetes Engine – Just another WordPress site'
+
+# This test should use finish with exit code 0 instead. However, make app/verify doesn't create all
+# resources that are created during real deployment. In this case it's TLS Secret.
+# If the test fails, it means that an issue is fixed and we should replace equals code from 1 to 0.
+- name: Site address should be 200 OK (external IP) (TLS)
+  bashTest:
+    script: curl -f -k "https://${EXTERNAL_IP"
```

Comment:
script: curl -f -k "https://${EXTERNAL_IP}"

Reply:
Good catch, thank you.